packages/fether-electron/src/main/index.js

```diff
@@ -4,12 +4,20 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 import electron from 'electron';
+import parseUrl from 'parse-url';
+// https://webpack.electron.build/development
+import sourceMapSupport from 'source-map-support';
```

Comment:
Do we need this? The docs you referenced only mentions installing the package afaict

Reply:
Yes, you're correct, I've pushed a commit that removes the unnecessary stuff. We only need to add it to package.json and the source maps still work.